README.md

```diff
@@ -0,0 +1,173 @@
+    "filters": [
+      {
+        "func": "https://raw.githubusercontent.com/OpendataDeveloperNetwork/ODEN-Transmogrifiers/ms-ss-rg-collector-update/collectors/collector-json.js"
+      },
+      {
+        "func": "https://raw.githubusercontent.com/OpendataDeveloperNetwork/ODEN-Transmogrifiers/dev/filters/validate.js",
+        "params": {
+          "validator": "json"
+        }
+      },
+      {
+        "func": "https://raw.githubusercontent.com/OpendataDeveloperNetwork/ODEN-Transmogrifiers/dev/filters/stringify.js",
+        "params": {
+          "indent": true
+        }
+      }
+    ],
+    "sinks": [
+      {
+        "func": "file_write",
+        "params": {
+          "path": "./all-my-data-1.json"
+        }
+      },
+      {
+        "func": "file_write",
+        "params": {
+          "path": "./all-my-data-2.json"
+        }
+      }
+    ],
+    "entries": [
+      {
+        "source": {
+          "func": "url_read",
+          "params": {
+            "path": "https://opendata.vancouver.ca/api/explore/v2.1/catalog/datasets/public-art/exports/json?lang=en&timezone=America%2FLos_Angeles"
+          }
+        },
+        "filters": [
+          {
+            "func": "https://raw.githubusercontent.com/OpendataDeveloperNetwork/ODEN-Transmogrifiers/dev/filters/canada/british-columbia/vancouver/public-art-json-to-json.js",
+            "params": {
+              "library": "https://raw.githubusercontent.com/OpendataDeveloperNetwork/ODEN-Transmogrifiers/dev/libraries/standard.js"
+            }
+          },
+          {
+            "func": "https://raw.githubusercontent.com/OpendataDeveloperNetwork/ODEN-Transmogrifiers/dev/filters/stringify.js",
+            "params": {
+              "indent": true
+            }
+          }
+        ],
+        "sinks": [
+          {
+            "func": "file_write",
+            "params": {
+              "path": "./my-vancouver-data-1.json"
+            }
+          },
+          {
+            "func": "file_write",
+            "params": {
+              "path": "./my-vancouver-data-2.json"
+            }
+          }
+        ]
+      },
+      {
+        "source": {
+          "func": "url_read",
+          "params": {
+            "path": "https://data.austintexas.gov/resource/uuk6-933w.json"
+          }
+        },
+        "filters": [
+          {
+            "func": "https://raw.githubusercontent.com/OpendataDeveloperNetwork/ODEN-Transmogrifiers/dev/filters/united-states/texas/austin/public-art-json-to-json.js",
+            "params": {
+              "library": "https://raw.githubusercontent.com/OpendataDeveloperNetwork/ODEN-Transmogrifiers/dev/libraries/standard.js"
+            }
+          }
+        ]
+      }
+    ]
+  }
+]
+```
+
+### How to Run a Manifest with the Transmogrifier
+
+To run the transmogrifier, you need an executor file, such as this [example](https://github.com/transmogrifier-io/transmogrifier/blob/dev/example-main.js). 
```

Comment:
should mention that the executor file simply reads and parses the manifest and passes it to the transmogrify function